### DISCUSSION TITLE
Configuration.yaml not found

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#890] [HUB: Home Assistant]
Ciao a tutti.
Questa mattina senza preavviso mi sono ritrovato con molti errori. Non ho fatto nessuna modifica ma non mi carica più il supervisor, gli add on ecc.
In più non mi trova più il configuration.yaml

![image](https://user-images.githubusercontent.com/72693930/137693900-13e48d2b-7da9-4978-a093-1815dbeaf638.png)
![image](https://user-images.githubusercontent.com/72693930/137693929-6054cd4b-2da1-4a04-9d8c-e0b5e93b7973.png)

Mi sapete dare qualche dritta? 
Posso semplicemente ripristinare il backup di ieri?

**Comment by Moxser** [comment]
Aggiungo anche questo screenshot magari puó aiutare.
![image](https://user-images.githubusercontent.com/72693930/137716935-b0db07c7-f94b-49a2-9041-bd0e534d902f.png)

**Reply by Moxser** [reply]
Alla fine ho deciso di re-installare tutto da 0 e ripristinato il backup di ieri. Cambiato SD con una nuova ma quello non era il problema di base. La vecchia SD l'ho controllata e non ha errori di alcun genere, il pc la legge benissimo. In ogni caso l'ho formattata e preparata per un eventuale nuova installazione.
La cosa che vorrei capire, se qualcuno ha qualche idea, come i file configuration.yaml si sia perso senza nessun motivo apparente.
Grazie a tutti

**Reply by snakuzzo** [reply]
Nessuna idea... quel file da solo non scompare.
Probabilmente avrai fatto qualche errore.

**Reply by Moxser** [reply]
Assolutamente d'accordo con te Diego.
Ma non ho fatto nessun cambiamento, nessuna integrazione nuova, nessuna automazione e nessuna nuova scrittura sul file di configurazione ecc. Inoltre sono stato all'esterno tutto il weekend e il lunedì mattina mi sono ritrovato nella situazione che vi ho descritto.
Secondo te posso fare dei controlli più approfonditi sulla SD che montavo prima?

**Reply by snakuzzo** [reply]
non ne ho idea...ma è 100 % errore umano, i file non scompaiono così
